lib/msf/ui/console/command_dispatcher/db/analyze.rb

```diff
@@ -61,10 +66,10 @@ def cmd_analyze(*args)
     suggested_modules
   end
 
+  def cmd_analyze_tabs(_str, _words)
+    return [] unless framework.db.active
```

Comment:
Could this just be `return [] active?`

Reply:
I opted not to use `active?` because it prints, which is bad for tab completion.